src/beacon-api/app/api/v1/webrtc/session_controller.py

```
@@ -0,0 +1,72 @@
+@router.delete("/sessions/{id}", response_model=WebRTCSessionResponse)
+async def close_session(
+    id: UUID,
+    db: AsyncSession = Depends(get_async_db),
+    user_id: str = Depends(get_current_user_id)
+):
+    """Close/end an active WebRTC session."""
+    return await session_service.close_session(db, id, user_id)
+
+@router.get("/sessions/{id}", response_model=WebRTCSessionResponse)
+async def get_session(
+    id: UUID,
```

Ruff (0.15.20)

[error] Function argument `id` is shadowing a Python builtin

(A002)